__tests__/api/utils.test.js

```diff
@@ -32,44 +49,77 @@ describe('Utils', () => {
     });
   });
 
+  describe('createReverseRelationships()', () => {
+    const state = {
+      resources: {
+        people: {
+          9: {
+            relationships: {
+              comments: {
+                data: [
+                  { id: '12', type: 'comments' }
+                ]
+              }
+            }
+          }
+        },
+        comments: {
+          5: {},
+          12: {}
+        },
+      }
+    };
+
+    test('updates hasMany relationships', () => {
+      const newState = imm(state);
+      utils.createReverseRelationships(state, newState, comments[0]);
+      const subject = newState.value().resources.people;
+
+      const expectedData = [{ id: '12', type: 'comments' }, { id: '5', type: 'comments' }];
+      expect(subject['9'].relationships.comments.data).toEqual(expectedData);
```

Comment:
may be useful to snapshot the subjects relationships on top of this expectation so we're notified if we mess the output